packages/lib-classifier/src/store/WorkflowStore/Workflow/Workflow.js

```diff
@@ -77,6 +79,14 @@ const Workflow = types
       })
 
       return anyTranscriptionTasks
+    },
+
+    get usesFreehandLineTool() {
+      const anyFreehandLineTool = self.tasks && Object.values(self.tasks).some(task => {
+        return task.tools.some(tool => tool.type === 'freehandLine')
+      })
```

Comment:
```suggestion
      const anyFreehandLineTool = self.tasks && Object.values(self.tasks)
.filter(task => !!task.tools)
.some(task => task.tools.some(tool => tool.type === 'freehandLine'))
```
This will error for tasks that don't have tools eg. question tasks. 

I'm not sure about the readability of the nested anonymous closures here.